### ISSUE TITLE
Cannot conditionally render specific elements

### ISSUE THREAD

**Issue by spiffytech** [forgojs/forgo#12]
I'm trying to render certain elements conditionally. With other frameworks I would render `null` like so:

```tsx
render() {
  let error: string | null = null;

  return (
    <div>
      {error ? <p>Encountered an error: {error}</p> : null}
    </div>
  );
}
```

Forgo blows up when I try to do that. I'm handling it by rendering an empty `div` for the moment, but I'd like a way to omit elements without adding noise to my DOM.

**Comment by jeswin** [comment]
Good point. Fix is coming.

**Comment by jeswin** [comment]
This is fixed in 0.0.27. Please close if it functions as expected.

**Comment by spiffytech** [comment]
Verified, thanks!
